### ISSUE TITLE
Roslyn Formatter support

### ISSUE THREAD

**Issue by mihaipopescu** [johnmcfarlane/unformat#22]
Would be interesting to add support for this formatter https://github.com/dotnet/codeformatter.

**Comment by johnmcfarlane** [comment]
Certainly. Provided the configuration can be generated and randomly tweaked
automatically, it should be a candidate.
